spec/acceptance/suites/default/87_disable_fips_spec.rb

```diff
@@ -40,7 +40,7 @@
         ensure_ssh_connection(agent)
 
         retry_on(agent, 'puppet agent -t',
-          :desired_exit_codes => [0],
+          :desired_exit_codes => [0,2],
```

Comment:
Why was this changed to allow exit code 2 (changes detected)?

Reply:
In case auditd toggles (it does sometimes)

Reply:
It's not an idempotency check so it doesn't matter.

Reply:
IMHO  it would be better to leave the system in a stable state for the next test.

Reply:
All of these tests were pushed to the very end of the stack because, by their nature, they may cause issues with system stability.

The final test that shoves everything back into a compliant mode was not modified and should achieve stability.